### PR TITLE
Bug 1209087 - Made use of Marionette's skip_if_e10s decorator instead of our own one

### PR DESCRIPTION
**Pull request by Motwani**
Made use of Marionette's skip_if_e10s decorator and removed duplicated code.

Made changes to files namely:
1) firefox_ui_harness/decorators.py
2) firefox_ui_tests/functional/private_browsing/test_about_private_browsing.py
3) firefox_ui_tests/functional/security/test_ssl_status_after_restart.py

Please review.

### CODE REVIEW

**Comment by whimboo** [comment]
Hi @Motwani, the changes as done here look great. I don't see anything which is missing. The only thing you should do now is to rebase your branch against upstream/master given that there are merge conflicts. Once that has been done I can merge your PR.

FYI it is helpful not to directly work on the master branch but to create individual development branches. That way you cannot really destroy master, but can also work on several issues at the same time.

Thank you for working on this bug!

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,4 @@
 _build/
 dist/
 venv/
+*.patch
```
Comment (whimboo) [inline .gitignore]: This change is not related to the bug you are working on. Please remove it. Maybe you want to add it to your local global .gitignore file.

**Comment by Motwani** [comment]
Removed the extra line from the file.

**Comment by whimboo** [comment]
Looks great to me now. I will squash the commits and land as a single changeset on mozilla-central in a moment.

**Comment by whimboo** [comment]
I also had to fix a merge conflict in the about private browsing test while rebasing this PR against mozilla-central. Merged PR as https://github.com/mozilla/firefox-ui-tests/commit/7c156f07841dc6c549bd1a230d31f5643823d9e4.
